website/src/plugins/rsdoctor/RsdoctorPlugin.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {PluginConfig} from '@docusaurus/types';
+
+async function createRsdoctorBundlerPlugin() {
+  // TODO Shitty workaround to bypass lib typechecking
+  //  package does not work will with skipLibCheck false
+  // eslint-disable-next-line
+  const {RsdoctorWebpackMultiplePlugin} = require('@rsdoctor/webpack-plugin');
```

Comment:
Not fan of this but will merge the PR as-is until the Rsdoctor maintainer gives me feedback